gems/sorbet/lib/gem-generator-tracepoint/tracer.rb

```diff
@@ -147,8 +152,12 @@ def self.install_tracepoints
             begin
               tp.disable
 
-              singleton = tp.method_id == :singleton_method_added
+              singleton = method_id_sym == :singleton_method_added
               receiver = singleton ? Sorbet::Private::RealStdlib.real_singleton_class(tp.self) : tp.self
+
+              # JRuby the main Object is not a module
```

Comment:
```suggestion
              # JRuby the main Object is not a module
              # so lets skip it, otherwise RealStdlib#real_instance_methods raises an exception since it expects one.
```